### PR TITLE
Fix wrong kcat indexing

### PR DESCRIPTION
**Pull request by carrascomj**
We pass a new mapping from "edges" to "enzyme reactions" to index kcats on stan.

We have to decide if we should tackle now phosphorylation, which is currently done by edge, where it should be done by enzyme and then expanded to the edge.

Checklist:

- [x] Updated any relevant documentation (none)
- [x] Add an adr doc if appropriate (none)
- [x] Include links to any relevant issues in the description (none)
- [x] Unit tests passing
- [ ] Integration tests passing

### CODE REVIEW

**Comment by carrascomj** [comment]
Please to a merge squash because I started from the wrong commit.

**Comment by carrascomj** [comment]
Failing because of safety issue on Jinja2 (development dependency, probably of sphinx).

Otherwise, should be good to go:

File: maud/getting_stan_inputs.py [hunk 1]
```diff
@@ -133,6 +133,7 @@ def get_network_properties_input(
     metabolite_codes = codify_maud_object(kinetic_model.metabolites)
     mic_codes = codify_maud_object(mics)
     enzyme_codes = codify_maud_object(enzymes)
+    ers_codes = codify_maud_object(kinetic_model.ers)
```
Comment (teddygroves) [inline maud/getting_stan_inputs.py:136]: nitpick: could we call this `er_codes`?

File: maud/getting_stan_inputs.py [hunk 2]
```diff
@@ -177,6 +178,10 @@ def get_network_properties_input(
         enzyme_codes[er.enzyme_id] if isinstance(er, EnzymeReaction) else 0
         for er in edges
     ]
+    edge_ers_code = [
```
Comment (teddygroves) [inline maud/getting_stan_inputs.py:181]: Another nitpick: could this change to `edge_er_code`?

**Review by teddygroves — APPROVED** [review]
Looks good, just a small name-related nitpick

**Pull request merged**